cmd/finch-daemon/main.go

```diff
@@ -66,14 +73,22 @@ func main() {
 	}
 	rootCmd.Flags().StringVar(&options.socketAddr, "socket-addr", defaultFinchAddr, "server listening Unix socket address")
 	rootCmd.Flags().BoolVar(&options.debug, "debug", false, "turn on debug log level")
-	rootCmd.Flags().IntVar(&options.socketOwner, "socket-owner", -1, "Uid and Gid of the server socket")
+	rootCmd.Flags().IntVar(&options.socketOwner, "socket-owner", 0, "Uid and Gid of the server socket")
 	rootCmd.Flags().StringVar(&options.debugAddress, "debug-addr", "", "")
 	rootCmd.Flags().StringVar(&options.configPath, "config-file", defaultConfigPath, "Daemon Config Path")
 	rootCmd.Flags().StringVar(&options.pidFile, "pidfile", defaultPidFile, "pid file location")
 	rootCmd.Flags().StringVar(&options.regoFilePath, "rego-file", "", "Rego Policy Path (requires --experimental flag)")
 	rootCmd.Flags().BoolVar(&options.skipRegoPermCheck, "skip-rego-perm-check", false, "skip the rego file permission check (allows permissions more permissive than 0600)")
 	rootCmd.Flags().BoolVar(&options.enableExperimental, "experimental", false, "enable experimental features")
 
+	// Setup credential helper addr to default.
```

Comment:
nit: this comment isn't that useful (but also grammar). Consider removing it
```suggestion
	// Set credential helper addr to default.
```